pkg/cmd/pipelinerun/delete_test.go

```diff
@@ -266,7 +266,7 @@ func TestPipelineRunDelete(t *testing.T) {
 			input:       seeds[4].pipelineClient,
 			inputStream: nil,
 			wantError:   false,
-			want:        "All but 2 PipelineRuns deleted in namespace \"ns\"\n",
+			want:        "All but 2 PipelineRuns(Completed) deleted in namespace \"ns\"\n",
```

Comment:
@piyush-garg @vinamra28 
how about this  `(Completed)`

Reply:
@pradeepitm12 there can be a case where the pipeline is in error state so should we show the same for that as well?

Reply:
Completed mean both to me - either success or failure, but it has completed running

Reply:
cc @vdemeester @chmouel Any thoughts for this?

Reply:
Maybe "Finished" but it's nitpick...

either I am not so fond of having another switch and instead would rather have a filter capability...

i.e:

```
--ignore-filter running,label=key,anotherlabel=another=key,failed,completed
```

which would ignore all of this

we would need to include a `--list-filter` too.

Reply:
As discussed in WG yesterday, is it fine to with current implementation and refactor later based on other use cases cc @vdemeester @chmouel